clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -125,6 +125,9 @@ def warn_float_compare_literal : Warning<
 def warn_double_const_requires_fp64 : Warning<
   "double precision constant requires %select{cl_khr_fp64|cl_khr_fp64 and __opencl_c_fp64}0, "
   "casting to single precision">;
+def warn_on_double_precision_use : Warning<
+  "double precision arithmetic used in device code; may cause reduced performance on GPU">,
+  InGroup<DoubleInDevice>;
```

Comment:
```suggestion
  InGroup<DiagGroup<"double-in-device">>;
```
You can define it inline like this instead.

Reply:
I didn't know we could do inline defining. TIL.